js/preload/passwordFill.js

```diff
@@ -101,7 +101,8 @@ function getInputs (names, types) {
 
     // We expect the field to have either 'name', 'formcontrolname' or 'id' attribute
     // that we can use to identify it as a login form input field.
-    if (checkAttribute(field, 'name', names) ||
+    if (field.type == "password" ||
+        checkAttribute(field, 'name', names) ||
```

Comment:
This would also return the password field for getUsernameFields() I think; it'd probably be better to remove this and just rewrite `getPasswordFields` to be:

```
return document.querySelectorAll('input[type=password]')
```

Reply:
> This would also return the password field for getUsernameFields()
Yeah I thought about that. 

Changing it to what you suggested seems better though.